### ISSUE TITLE
Not all speakers shown

### ISSUE THREAD

**Issue by mikosoft83** [punxaphil/custom-sonos-card#325]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**
7.6.0 but also 7.5.0

**Last working release (if known):**
I installed 7.5.0 and it was always like that

**Browser and Operating System:**
Edge, Windows 10

**Description of problem:**

I can't see all of my Sonos speakers in the card. They are available in the integration and show with other media cards.

![image](https://github.com/punxaphil/custom-sonos-card/assets/63317931/7effab7e-37e8-4383-b58c-281c0dbf217e)

**Comment by punxaphil** [comment]
It only show available devices.
Please "proove" that this device is available in your setup by a screenshot or similar. Go here: http://homeassistant.local/developer-tools/state

**Comment by mikosoft83** [comment]
Actually, I found the issue thanks to looking at the attributes in dev tools. Seems like the Kitchen player had group members defined but was not really grouped. When I regrouped and ungrouped all speakers it is now visible.
But I believe this actually happened when I tried grouping speakers using your card in the "no apply button" mode. For me, it actually doesn't work, when I click any speaker to group it, it briefly show the rotating circle and then does nothing, the speakers are not grouped at all. When using the apply button, everything works as expected.
